test/e2e/app-dir/import-trace-dynamic-eval/node_modules/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "version": "1.0.0",
```

Comment:
should be under `node_modules/foo`?

And we should add `"type": "module"` and `exports: "./index.js"` since we exported a ESM index file here